tests/rip7560/test_rip7560.py

```diff
@@ -232,6 +235,7 @@ def test_account_eth_sendTransaction7560_banned_opcode(
     assert state_before == 0
     tx_7560.sender = wallet_contract_rules.address
     tx_7560.signature = to_prefixed_hex(banned_op)
+    tx_7560.nonceKey = hex(0)
```

Comment:
why on each test? the whole idea of tx_7560 is to fill in all the default fields, and this is a default

Reply:
Removed.